magnum_cluster_api/resources.py

```diff
@@ -1709,6 +1711,8 @@ def apply_cluster_from_magnum_cluster(
 def get_kubeadm_control_plane(
     api: pykube.HTTPClient, cluster: magnum_objects.Cluster
 ) -> objects.KubeadmControlPlane:
+    if cluster.stack_id is None:
+        return None
```

Comment:
wouldn't the filter below return 0 results, and because it gave 0 results, it will not do the if statement and simply return None?